tests/dtm/test_single_process_restart.py

```diff
@@ -123,6 +123,12 @@ def test_read_during_m0d_restart(self):
             proc_read_op.join()
         resp = que.get()
         assert_utils.assert_true(resp[0], resp[1])
+        workload_info = resp[1]
+
+        self.log.info("Step 6: Perform Delete Operations on data written in Step 1:")
```

Comment:
Code changes will be added in setup_method to create buckets and hence the delete operation will delete only the objects and not the buckets